pgml-sdks/python/notebooks/extractive_question_answering.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/emmanuel/Documents/Work/Alba/Synced_projects/Alba_Waldo/waldo_postgresml/postgresml_alba/.devenv/state/venv/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
```

Comment:
You may want to clear notebook output, before merging them.

Reply:
Thanks. 

Still WIP. Back into not compiling to the end. I cannot figure out why Nix is not resolving some paths. 

Will sanitise the notebook, move to a separate folder and amend the docs once that works again.